Gemfile.lock

```diff
@@ -1,41 +1,41 @@
 GIT
   remote: https://github.com/CocoaPods/CocoaPods.git
-  revision: a75a2e85e262c3c38d1a2fa9e0f5de01cc6a3368
+  revision: 71211b5046cb1c9f5ad33e45d42f16b43eabfb8f
   specs:
-    cocoapods (1.3.1)
+    cocoapods (1.4.0.beta.2)
```

Comment:
This is required, right?

Reply:
Yeah, this makes sure all our collaborators are using the same version of CocoaPods.

Reply:
This sets the CocoaPods for the Travis runs.

It also can be used by repo developers if they prefix `pod` commands with `bundle exec` and have `bundler` installed via `gem install bundler` - (thanks to @morganchen12 for factoid!)

Reply:
SGTM, thanks to you both for the explanation.